.pre-commit-config.yaml

```diff
@@ -45,8 +45,7 @@ repos:
     exclude: >-
       ^docs/[^/]*\.svg$
   - id: requirements-txt-fixer
-    exclude: >-
-      ^requirements/.*\.txt$
+    files: requirements/.*\.in$
```

Comment:
@Dreamsorcerer honestly, I'd delete this formatter altogether. It's meant to make `pip freeze` output predictable. But our `.in` files are crafted manually, and I would rather structure them manually too. Besides, IIRC such hooks don't play well with comments in the input files.